concept/thing/attribute.rs

```diff
@@ -171,8 +178,13 @@ impl<'a> ThingAPI<'a> for Attribute<'a> {
         snapshot: &impl ReadableSnapshot,
         type_manager: &TypeManager,
     ) -> Result<Prefix, ConceptReadError> {
-        let value_type = type_.get_value_type(snapshot, type_manager)?.unwrap();
-        Ok(Self::Vertex::value_type_category_to_prefix_type(value_type.category()))
+        let value_type = type_.get_value_type(snapshot, type_manager)?;
+        match value_type {
+            // It should be valid to try getting a prefix without a value type, so we just return
+            // the first prefix of possible. Should get 0 instances anyway
+            None => Ok(Prefix::VertexAttributeBoolean),
```

Comment:
@flyingsilverfin Well, it was my intention to be able to use it for types without value types, because it is a valid request to get instances of abstract types. I might change the caller's code not to use this method, but, if I remember correctly, the code was generic and I didn't want to change anything there. Might do.

Reply:
Rewritten to error if there is no value + add filtering on the caller side